services/wallet/api.go

```diff
@@ -488,31 +488,6 @@ func gweiToWei(val *big.Float) *big.Int {
 	return res
 }
 
-func (api *API) GetSuggestedRoutes(
```

Comment:
Just mark it as deprecated and return an error for now. And remove the function when the clients are updated.
No need for such destructive actions 😄

Reply:
But I wanted to clean the api, not to keep things around, that's why we added the same names with V2 suffixes, to not make breaking changes. Also I plan to sync with @alwx (actually we already chat in the wallet channel) on this and don't plan to merge this before mobile is ready for this change, then we will sync and merge changes at the same time.